lld/test/ELF/invalid/broken-relaxation-x64.test

```diff
@@ -44,4 +44,132 @@ Symbols:
     Value:    0x12345
     Size:     4
     Binding:  STB_GLOBAL
- 
+
+
+# RUN: yaml2obj --docnum=2 %s -o %t2.o
+# RUN: not ld.lld %t2.o -o /dev/null 2>&1 | FileCheck --check-prefix=ERR2 %s
+# ERR2: invalid prefix with R_X86_64_CODE_4_GOTTPOFF!
```

Comment:
We always test error: in error messages.

Reply:
Updated.